cartography/intel/lastpass/users.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import neo4j
+from dateutil import parser as dt_parse
+from requests import Session
+
+from cartography.client.core.tx import load_graph_data
+from cartography.graph.querybuilder import build_ingestion_query
+from cartography.models.lastpass.tenant import LastpassTenantSchema
+from cartography.models.lastpass.user import LastpassUserSchema
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+# Connect and read timeouts of 60 seconds each; see https://requests.readthedocs.io/en/master/user/advanced/#timeouts
+_TIMEOUT = (60, 60)
+
+
+@timeit
+def sync(
+    neo4j_session: neo4j.Session,
+    lastpass_provhash: str,
+    common_job_parameters: Dict[str, Any],
+) -> None:
+    users = get(lastpass_provhash, common_job_parameters)
+    formated_users = transform(users)
+    load(neo4j_session, formated_users, common_job_parameters)
```

Comment:
Oh, this is also missing a cleanup function. Just need to add one to cleanup the users. No need to clean up the tenant.

```python
def cleanup(neo4j_session: neo4j.Session, common_job_parameters: Dict[str, Any]) -> None:
    GraphJob.from_node_schema(LastpassUserSchema(), common_job_parameters).run(neo4j_session)
```